atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -424,9 +426,12 @@ def __init__(self, data, pulp_data, odcs_config, koji_tag=None, arches=None):
             for arch in pulp_data or {}:
                 if arch in self.arches:
                     self.pulp[arch] = pulp_data[arch]
-            self.flags = None
+            self.flags = []
```

Comment:
This changes the way we interact with ODCS from "only passing the flags param when not empty" to "always passing a flags param". Going through ODCS code, this does not seem to be an issue.